packages/arb-validator/rollup/nodeGraph.go

```diff
@@ -236,18 +236,26 @@ func (chain *NodeGraph) CommonAncestor(n1, n2 *Node) *Node {
 	return n1.prev
 }
 
-func (chain *NodeGraph) generateNodePruneInfo() []pruneParams {
+func (chain *NodeGraph) generateNodePruneInfo(stakers *StakerSet) []pruneParams {
```

Comment:
Why does this function take `stakers` rather than using  `chain.stakers`?

Reply:
Because chain is a NodeGraph, which doesn't have a stakers field.  (stakers is a field of StakedNodeGraph, which is a "superclass" of NodeGraph.)

Reply:
Ah right. So then probably it should be moved into `StakedNodeGraph`

Reply:
Did that.